app/javascript/shared/http.ts

```diff
@@ -33,11 +33,18 @@ if (csrfToken) {
 }
 
 const kyApi = ky.extend({
+  headers: {
+    'Content-Type': 'application/json',
+  },
   hooks,
   throwHttpErrors: false,
 });
 
 export const http = {
+  async delete<T>(url: string) {
```

Comment:
I chose not to include a payload in the method signature. We could add that, in the future, if needed. However, I think it's uncommon for DELETE requests to have a payload, so I think we might never need it.